agent/cita_exporter/Dockerfile.arm64

```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:18.04
+
+WORKDIR /config
+
+RUN apt-get update && apt-get install -y  \
+	  python3 \
+	  python3-pip \
+	  libcurl4-openssl-dev \
+    && apt-get clean \
+    && apt-get autoclean \
+    && rm -rf /var/lib/apt/lists/*
+
+ADD ./cita-cli-arm64 /bin/cita-cli
```

Comment:
I think we can put this bin into a docker image(compare to https://hub.docker.com/_/cita-cli/), then copy it in Dockefile